src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/ExtractRequest.cs

```diff
@@ -26,6 +26,11 @@ public class ExtractParams : DacFxParams
         /// Gets or sets the target for extraction
         /// </summary>
         public DacExtractTarget ExtractTarget { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether permissions should be included in the extract
+        /// </summary>
+        public bool? ImportPermissions { get; set; }
```

Comment:
nit: Since Import has its own meaning in DacFx and this is used with Extract, `IncludePermissions` may be a better name?

Reply:
makes sense - updated!